### PR TITLE
Adds a script to anon data, adds anoned FA18 data

### PR DESCRIPTION
**Pull request by JParisFerrer**

### CODE REVIEW

File: test_data/sections/_meetings.csv [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name,ci_term,ci_name,ci_year
+lab_huffman,Fall,CS 225,2018
+lab_ml,Fall,CS 225,2018
+lab_btree,Fall,CS 225,2018
+Lab_avl,Fall,CS 225,2018
+lab_inheritance,Fall,CS 225,2018
+Lab_hash,Fall,CS 225,2018
```
Comment (Binary-Eater) [inline test_data/sections/_meetings.csv]: Why is this one capitalized?

File: test_data/sections/_sectionMeetings.csv [hunk 2]
```diff
@@ -0,0 +1,17 @@
+m_name,s_name,ci_term,ci_name,ci_year
+lab_huffman,AYP,Fall,CS 225,2018
+lab_ml,AYL,Fall,CS 225,2018
+lab_btree,AYN,Fall,CS 225,2018
+Lab_avl,AYH,Fall,CS 225,2018
+lab_inheritance,AYC,Fall,CS 225,2018
+Lab_hash,AYI,Fall,CS 225,2018
```
Comment (Binary-Eater) [inline test_data/sections/_sectionMeetings.csv]: Same here

File: test_data/sections/_meetings.csv (hunk 1 above)
Comment (JParisFerrer) [reply test_data/sections/_meetings.csv]: messy data, I can fix

File: test_data/sections/_sectionMeetings.csv [hunk 3]
```diff
@@ -0,0 +1,15 @@
+m_name,s_name,ci_term,ci_name,ci_year
+lab_huffman,AYV,Fall,CS 225,2018
```
Comment (JParisFerrer) [inline test_data/sections/_sectionMeetings.csv:179]: wait, these are all messed up lol. it's not crossproduct correctly

Comment (JParisFerrer) [reply test_data/sections/_sectionMeetings.csv:179]: and fixed

**Review by Binary-Eater — APPROVED** [review]

**Pull request merged**